components/fms-forwarder/Cargo.toml

```diff
@@ -30,6 +30,7 @@ readme.workspace = true
 
 [dependencies]
 async-trait = { workspace = true }
+async-std = {workspace = true, features = ["attributes"] }
```

Comment:
do we still need this?

Reply:
async-std removed from Cargo.toml files.